scripts/entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set +ex
+
+ecosystem_list=("openshift" "knative" "kubevirt")
+
+# Run the data collector for given eco systems and no of days to colelct data
+python3 app/run_data_collector.py -eco-systems ${ecosystem_list[*]} -days ${DAYS};
```

Comment:
Usual convention is to use double hyphen for expanded flag name, and single hyphen for single char flag name.

e.g. --ecosystem or -e

Reply:
corrected